test/Concurrency/transfernonsendable.swift

```diff
@@ -1997,3 +1997,13 @@ func localCaptureDataRace3() async {
 
   print(x)
 }
+
+nonisolated func localCaptureDataRace4() {
+  var x = 0
+  _ = x
+
+  Task.detached { @MainActor in x = 1 } // expected-tns-warning {{sending 'x' risks causing data races}}
```

Comment:
might be worth having a test that crosses isolation through a 'normal' `Task.init` as well (unless that's also covered elsewhere)

Reply:
that won't matter for what this is testing.